icicle/appUtils/msm/msm.cu

```diff
@@ -554,7 +564,10 @@ void bucket_method_msm(
       NUM_THREADS = min(MAX_TH, s);
       NUM_BLOCKS = (s + NUM_THREADS - 1) / NUM_THREADS;
       single_stage_multi_reduction_kernel<<<NUM_BLOCKS, NUM_THREADS, 0, stream2>>>(
-        large_buckets, large_buckets, s * 2, 0, 0, 0);
+        large_buckets, large_buckets, s * 2, 0, 0, 0, s);
+
+      CHECK_LAST_CUDA_ERROR();
+      CHECK_SYNC_DEVICE_ERROR();
```

Comment:
This line adds extra synchronisation which wasn't here before. Why is this needed?

Reply:
forgot to remove for debugging, i wanted to just leave the CHECK_LAST_CUDA_ERROR

thanks!